components/cli/pkg/util/utils.go

```diff
@@ -514,3 +522,86 @@ func GetFileSize(path string) (int64, error) {
 
 	return file.Size(), nil
 }
+
+func RenameFile(oldName, newName string) error {
+	err := os.Rename(oldName, newName)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func ReplaceFile(fileToBeReplaced, fileToReplace string) error {
```

Comment:
The function name should start with a simple letter. Please fix the below functions as well.